src/porepy/grids/mortar_grid.py

```diff
@@ -619,11 +563,7 @@ def _convert_to_vector_variable(
         dimension is 1 (default for all the above methods), the projection matrix
         will in effect not be altered.
         """
-        if nd == 1:
```

Comment:
Doing the Kronecker product takes a lot of time, even if nd==1. Revert.

Reply:
yes